.github/workflows/update-lock-files-pr.yml

```diff
@@ -6,12 +6,12 @@ on:
 
 permissions:
   contents: write
+  statuses: write
 
 jobs:
   update-lock-files:
     if: >-
-      github.repository == 'scikit-learn/scikit-learn'
-      && github.event.issue.pull_request
+      github.event.issue.pull_request
```

Comment:
Anyone who reviews this please add a "Request changes" to this so that we don't forget to revert this again as we did last time 🤣 This was purely for testing purpose on my fork.